shell/platform/darwin/common/buffer_conversions.mm

```diff
@@ -7,22 +7,22 @@
 #include "flutter/fml/macros.h"
 #include "flutter/fml/platform/darwin/scoped_nsobject.h"
 
+static_assert(__has_feature(objc_arc), "ARC must be enabled.");
+
 namespace flutter {
 namespace {
 class NSDataMapping : public fml::Mapping {
  public:
-  explicit NSDataMapping(NSData* data) : data_([data retain]) {}
+  explicit NSDataMapping(NSData* data) : data_(data) {}
 
-  size_t GetSize() const override { return [data_.get() length]; }
+  size_t GetSize() const override { return [data_ length]; }
 
-  const uint8_t* GetMapping() const override {
-    return static_cast<const uint8_t*>([data_.get() bytes]);
-  }
+  const uint8_t* GetMapping() const override { return static_cast<const uint8_t*>([data_ bytes]); }
```

Comment:
`data_.bytes`

Modernize ALL the things!

Reply:
Using the shiny new[1] syntax!

[1] for definitions of _new_ in excess of a decade.